### PR TITLE
Feat : Filter Functionality

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Feat Filter Functionality
## Added 
- filter by labels
- filter by dateandtime
- added date and time in note card

### CODE REVIEW

**Review by sharmayatendra — APPROVED** [review]
code looks clean
design is consistent
folder structure is good

File: src/Components/LabelModel/LabelModel.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    if (fromNoteFeild) {
+      setNotes((prev) => ({ ...prev, label: e.target.value }));
+    } else {
+      const resp = await editNotesService(
+        { ...notes, label: e.target.value },
+        encodedToken
+      );
+      noteDispatch({
+        type: "UPDATE_NOTE",
+        payload: resp.data.notes,
```
Comment (devdeadviz) [inline src/Components/LabelModel/LabelModel.jsx:23]: You can destructure it more before sending it here.

File: src/Pages/LabelPage/LabelPage.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      toast.error("Label already exists");
+      setLabelValue("");
+    }
+  };
+  const deleteTableHandler = (labelName) => {
+    const data = label.filter((item) => item !== labelName);
+    noteDispatch({ type: "REMOVE_LABEL", payload: data });
+  };
+  return (
+    <>
```
Comment (devdeadviz) [inline src/Pages/LabelPage/LabelPage.jsx:31]: Good use of react fragment.

**Review by devdeadviz — APPROVED** [review]
- The code is clean.
- Good file structure.
- You have followed all the best practices.
- Good work
- Good use of barrel exports.

**Pull request merged**